packages/fxa-auth-server/lib/metrics/context.js

```diff
@@ -299,7 +298,7 @@ module.exports = function (log, config) {
     );
     const flowSignatureBytes = Buffer.from(flowSignature, 'hex');
     const expectedSignatureBytes = calculateFlowSignatureBytes(metadata);
-    if (!bufferEqualConstantTime(flowSignatureBytes, expectedSignatureBytes)) {
+    if (!crypto.timingSafeEqual(flowSignatureBytes, expectedSignatureBytes)) {
```

Comment:
Looks right, https://nodejs.org/api/crypto.html#cryptotimingsafeequala-b.

This block of code isn't too risky since it is used check metrics context.